bookdown/01-Introduction.Rmd

```diff
@@ -43,44 +48,43 @@ code, or a target's dependencies have changed (or become "outdated" in {targets}
 speak).
 
 **lakeSR** is broken down into functional groups of targets, listed below with a
-brief summary about what task(s) each group completes.
```

Comment:
"functional group" == "a list of functions performed in specific order"?

Reply:
kind of... more like functions that are grouped together to perform a specific task.

Edited to: 

lakeSR is broken down into _groups of targets that perform specific tasks_, listed below with a brief summary about what task(s) each group completes. This particular workflow incorporates both R and Python programming languages to complete these tasks.